samples/expo/metro.config.js

```diff
@@ -15,8 +15,8 @@ config.watchFolders.push(path.resolve(__dirname, '../../node_modules/@sentry'));
 config.watchFolders.push(path.resolve(__dirname, '../../node_modules/@sentry-internal'));
 config.watchFolders.push(path.resolve(__dirname, '../../node_modules/tslib'));
 config.watchFolders.push(path.resolve(__dirname, '../../node_modules/hoist-non-react-statics'));
-config.watchFolders.push(path.resolve(__dirname, '../../node_modules/localforage'));
 config.watchFolders.push(path.resolve(__dirname, '../../node_modules/@react-native/js-polyfills'));
+config.watchFolders.push(path.resolve(__dirname, '../../node_modules/preact'));
```

Comment:
This is hopefully not needed anymore, could you verify? We changed some things late in the v8 cycle there!

Reply:
Yes, I've removed it, not needed any more.